clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/UpsertGoogleImageTagsAtomicOperationUnitSpec.groovy

```diff
@@ -117,9 +117,9 @@ class UpsertGoogleImageTagsAtomicOperationUnitSpec extends Specification impleme
       }
 
     then:
-      1 * imagesMock.setLabels(PROJECT_NAME, IMAGE_NAME, { globalSetLabelsRequest = it }) >> setLabelsMock
+      1 * imagesMock.setLabels(_, _, _) >> setLabelsMock
       1 * setLabelsMock.execute()
-      globalSetLabelsRequest.labels == TAGS
+//      globalSetLabelsRequest.labels == TAGS
```

Comment:
? if it's not needed, let's delete this and line 193

Reply:
I am not sure if this is okay or not. I'd prefer for someone like @duftler to give this a sanity check. That's why they're commented out

Reply:
I think we can refactor the test to re-enable that checking, but don't want to block the boot2 merge on it.  You can `//TODO(ezimanyi)` or I'll just take a look in the next few days either way

Reply:
👍 thanks @ezimanyi! I'll just leave as is.

Reply:
Fixed in spinnaker/clouddriver#3657